airbyte-integrations/bases/base-normalization/entrypoint.sh

```diff
@@ -113,9 +113,20 @@ function main() {
     . /airbyte/sshtunneling.sh
     openssh "${PROJECT_DIR}/ssh.json"
     trap 'closessh' EXIT
+
+    # We don't run dbt 1.0.x on all destinations (because their plugins don't support it yet)
+    # So we need to only pass `--event-buffer-size` if it's supported by DBT.
+    dbt --help | grep -E -- '--event-buffer-size'
```

Comment:
@edgao: If the bundled DBT version in is not supporting the --event-buffer-size flag, the script will exit here because of the previous `set -e` statement.

This is the case with the MySQL normalization container (normalization-mysql:0.1.74) at the moment, likely the reason why nornalization with MySQL destination currently fails

Reply:
Correct, thanks for pointing that out!

We should probably use this `set +e` option before calling that line
https://github.com/airbytehq/airbyte/blob/9f21fae6684e6833e8896d017cd5c859046b61a7/airbyte-integrations/bases/base-normalization/entrypoint.sh#L36

Reply:
opened #11578